index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Carrie Short - Web Developer</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link type="text/css" rel="stylesheet" href="vendor/styles/font-awesome.min.css">
```

Comment:
Consider using ico moon instead of font awesome as it's a standard for code fellows, and it'd be best if you learn it now instead of during project week.